supply/lib/supply/uploader.rb

```diff
@@ -115,6 +115,15 @@ def upload_binaries
         apk_version_codes.push(upload_binary_data(apk_path))
       end
 
+      mapping_paths = [Supply.config[:mapping]] unless (mapping_paths = Supply.config[:mapping_paths])
+      mapping_paths.zip(apk_version_codes).each do |mapping_path, version_code|
+        puts mapping_path
```

Comment:
Can we remove the `puts` here? 

If you think the output is valuable, provide a nicely formatted string like `UI.message "Uploading #{mapping_path} for #{version_code}."`